go/kbnm/windows/kbnm.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Another option: We could put it ingo/kbnm/host_json.windows (or something right next to the original template) and then copy it over during the `build_prerelease.cmd` phase. At least this way it'll be all in one place-ish and we'll remember to keep them in sync.